Models/Models.js

```diff
@@ -41,7 +41,17 @@ exports.festchArticleCommentsById = (article_id) => {
         return result.rows
     })
 }
-exports.fetchAllArticles = () => {
+exports.fetchAllArticles = (topic) => {
+    if(topic){
+        return db.query('SELECT * FROM articles WHERE topic = $1;',
+        [topic])
+        .then((result) => {
+            if (result.rows.length === 0) {
```

Comment:
Unfortunately this is another situation where sometimes an empty array is okay...
The `paper` topic in the test data has no articles attached to it, and that would be a `200` and an empty array.
We only want `404` if the topic doesn't exist in the database